docs/products/databases/managed-databases/guides/postgresql-migrate/index.md

```diff
@@ -0,0 +1,62 @@
+
+    If you are connecting to a remote database, add `--host [host]` and `--port [port]` to the command above, replacing *[host]* with the host url of your remote database and *[port]* with the port number.
+
+    {{< note >}}
+This step requires admin/superuser access to the PostgreSQL cluster. If you do not have this level of access and you still wish to preserve the existing roles, you can manually create the roles on your Managed Database though [SQL](https://www.postgresql.org/docs/current/sql-createrole.html) or the [`createuser`](https://www.postgresql.org/docs/current/app-createuser.html) functionality of psql. If you do not do this, all data will be owned by the `linpostgres` user once you import your data to your Managed Database cluster.
+{{</ note >}}
+
+1.  **Export each database you wish to backup.** Replace *[database-name]* with the name of your database and *[user]* with the username for your source database. The other options in this command are used to ensure maximum compatibility with Linode's PostgreSQL Managed Databases.
+
+        pg_dump -Fd --quote-all-identifiers --verbose --lock-wait-timeout=480000 --no-unlogged-table-data --serializable-deferrable --jobs=1 --dbname [database-name] --username [user] --file database.backup
+
+    Again, add `--host [host]` and `--port [port]` to the command above if you are connecting to a remote database. You can run this command for each database you wish to export, though you may want to edit the `--file` option with a custom filename.
+
+## Import to the Target Managed Database
+
+Once you've successfully backed up the source database, you can import your data into your Managed Database (the *target* database). To import the database, this guide covers the [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) utility.
+
+1.  **Import your database roles to your Managed Database** using the psql command below. Replace *[host]*, and *[username]* with the corresponding values for your Managed Database. See [Connection Details](/docs/products/databases/managed-databases/guides/postgresql-connect/#view-connection-details).
+
+        psql --host=[host] --username=[username] --dbname=postgres --file=roles.backup
+
+    This assumes your database roles backup file is called *roles.backup* and is located in your current directory (as per previous steps in this guide). If you used a different filename or path for your backup, replace *roles.backup* as needed in the above command.
+
+1.  **Create your database**, making sure that your preferred database name doesn't already exist. If it does, you can drop the database before creating it. In the commands below, replace *[database-name]* with the name of your database and *[host]* and *[username]* with the corresponding [Connection Details](/docs/products/databases/managed-databases/guides/postgresql-connect/#view-connection-details) for your Managed Database.
```

Comment:
@wildmanonline Should we have a warning note here about how dropping a table will result in the loss of any data in that table